polaris-migrator/bin/migrator-cli.js

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env node
+
+const {run} = require('./../dist');
```

Comment:
Can we make this `"type": "module"` and remove require?

Reply:
I did give this a try and it lead to multiple other issues so punted for now due to time-to-importance ratio. We can certainly revisit later.

I did add an ESM build for consuming packages like the [Shopify CLI](https://github.com/Shopify/cli)